Make_It_Sense/cpx-cap-touch.py

```diff
@@ -0,0 +1,20 @@
+from adafruit_circuitplayground.express import cpx
+
+# NeoPixel blank out
+cpx.pixels.fill( (0, 0, 0) )
```

Comment:
You don't need the extra spaces around the `(0, 0, 0)`. It can be `((0, 0, 0))`.

Reply:
just to emphasize its a tuple

Reply:
Pylint isn't complaining, so we can leave it as-is, but it's not how we've done it previously. If you'd like to leave it, it's up to you. If you'd rather it be consistent, then consider changing it.